deepaas/cmd/execute.py

```diff
@@ -63,21 +72,40 @@
 # Loading the model installed
 
 
-def model_name():
-    global MODEL_NAME
+def get_model_name():
+    models = {}
+    models_loaded = False
+    model_name = CONF.model_name
     try:
         for name, model in loading.get_available_models("v2").items():
-            MODEL_NAME = name
+            models[name] = model
+            if model_name == name:
+                models_loaded = True
+                model_obj = model
+
+        if models_loaded is True:
```

Comment:
This will be always false, as models_loaded is a local variable.

I think that this function can be simplified as follows (code not checked):

    def get_model():
        models = loading.get_available_models()
        if CONF.model_name:
            model_obj = models.get(CONF.model_name)
            if model_obj is None_
                    (raise error)
            return CONF.model_name, model_obj
        elif len(models) == 1:
            return models.popitem()
        else:
            (raise error, as we have several models and no CONF_model_name is present)